Resources/Private/Translations/en/ValidationErrors.xlf

```diff
@@ -17,6 +17,10 @@
 				<source>This property is required</source>
 				<target xml:lang="en"></target>
 			</trans-unit>
+			<trans-unit id="1221560719" xml:space="preserve" approved="yes">
+        <source>The reCaptcha-Check failed.</source>
+        <target xml:lang="en"></target>
```

Comment:
Missing translation (or remove target).

Reply:
This should be fixed now, or I'm not so smart. 😄